app/views/courses/TeacherCoursesView.coffee

```diff
@@ -51,6 +56,17 @@ module.exports = class TeacherCoursesView extends RootView
     @paidTeacher = @paidTeacher or @prepaids.find((p) => p.get('type') in ['course', 'starter_license'] and p.get('maxRedeemers') > 0)?
     @render?()
 
+  fetchChangeLog: co.wrap ->
+    yield api.courses.fetchChangeLog()
+    .then((changeLogInfo) => 
+      @courses.models.forEach (course) =>
+        changeLog = _.filter(changeLogInfo, { 'c' : course.get('slug') })
+        @courseChangeLog[course.id] = _.mapValues(_.groupBy(changeLog, 'd'))
+    )
```

Comment:
Do we need to `@render?()` on successful fetch?

Reply:
Locally this probably returns very quickly, before the page is rendered.  However, this fetch could potentially take long enough in production to return after the page has rendered with an empty `@courseChangeLog`, after which returned logs would not be visible.  I'd suggest added a `@render?()` at the end of this `.then` handler to be sure.